makefile

```diff
@@ -208,7 +208,7 @@ docstring: dev-requirements
 
 
 test: requirements dev-requirements
-	$(PYTEST) $(args) $(TESTDIR)
+	$(PYTEST) $(args) $(TESTDIR)/vision/checks/distribution/image_dataset_drift_test.py
```

Comment:
what's not supposed to be here right?

Reply:
no, will delete. just to make me testing iterations faster